src/request-pipeline/xhr/headers.js

```diff
@@ -9,5 +9,6 @@ export default {
     corsSupported:           'x-hammerhead|xhr|cors-supported-header',
     withCredentials:         'x-hammerhead|xhr|with-credentials-header',
     origin:                  'x-hammerhead|xhr|origin',
```

Comment:
Rename to `x-hammerhead|xhr|cors-supported`, `x-hammerhead|xhr|with-credentials`.
Do this in the current pull request because it is very simple